.github/PULL_REQUEST_TEMPLATE/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,50 @@
+# Pull Request Template
+
+## Description
+
+Please provide a brief description of the change and mention the issue it addresses.Also, include relevant motivation and context. List any dependencies that are required for this change.
+
+Please delete options that are not relevant and please make sure that you X the appropriate boxes.
```

Comment:
[suggestion] It'd be best to comment out the instructions so that contributors can work find space to add this information. 
```suggestion
<!-- Please provide a brief description of the change and mention the issue it addresses.Also, include relevant motivation and context. List any dependencies that are required for this change.

Please delete options that are not relevant and please make sure that you X the appropriate boxes.
-->
```